core/debug/src/debug.c

```diff
@@ -30,6 +30,41 @@ uint8_t g_buffer[DEBUG_MAX_BUFFER];
 int g_buffer_pos;
 
 #ifndef CHANNEL_DEBUG
+#ifdef CORE_BEETLE
+#define UART1       0x40005000
+#define UART1DATA   (UART1)
+#define UART1STATE  (UART1 + 0x4)
+#define UART1CTRL   (UART1 + 0x8)
+#define UART1BDIV   (UART1 + 0x10)
+
+#define beetle_uart1_readl(reg) *(volatile uint32_t *)reg
+#define beetle_uart1_writel(reg, val)  *(uint32_t *)reg = val;
```

Comment:
@fvincenzo no good reason to use macros here: can you please use static inlines instead?